corporate/tests/test_stripe.py

```diff
@@ -2277,7 +2277,9 @@ def test_request_sponsorship(self) -> None:
             self.assertEqual(message.subject, "Sponsorship request (Open-source project) for zulip")
             self.assertEqual(message.reply_to, ["hamlet@zulip.com"])
             self.assertEqual(self.email_envelope_from(message), settings.NOREPLY_EMAIL_ADDRESS)
-            self.assertIn("Zulip sponsorship <noreply-", self.email_display_from(message))
+            self.assertIn(
+                f"{settings.SERVICE_NAME} sponsorship <noreply-", self.email_display_from(message)
```

Comment:
I think this shouldn't SERVICE_NAME, since the name is hard-coded?

Reply:
**Correct**
Updated the PR -- thanks.